recipes/csvmonkey/all/conanfile.py

```diff
@@ -0,0 +1,50 @@
+import os
+
+from conans.errors import ConanInvalidConfiguration
+from conans import ConanFile, CMake, tools
+
+required_conan_version = ">=1.36.0"
+
+class CSVMONEKYConan(ConanFile):
+    name = "csvmonkey"
+    license = "BSD-3-Clause"
+    url = "https://github.com/conan-io/conan-center-index"
+    description = "Header-only vectorized, lazy-decoding, zero-copy CSV file parser "
+    topics = ("csv parser", "header only", "vectorized")
```

Comment:
```suggestion
    topics = ("csv-parser", "header-only", "vectorized")
```

Because a search engine can consider space character as multiple words.